QEP-auxiliary-storage.rst

```diff
@@ -0,0 +1,178 @@
+.. _qep#[.#]:
+
+========================================================================
+QGIS Enhancement ??: Auxiliary storage
+========================================================================
+
+:Date: October 2015
+:Author: Hugo Mercier
+:Contact: hugo dot mercier at oslandia dot com
+:Last Edited: 
+:Status:  
+:Version:
+
```

Comment:
I like the general direction this proposal is going... it sounds a lot like how MapInfo embeds custom labeling properties within a workspace file, but without all the horrible performance issues MI's implementation suffers from.